lib/pure/distros.nim

```diff
@@ -141,8 +141,8 @@ template unameRelease(cmd, cache): untyped =
     cache = (when defined(nimscript): gorge(cmd) else: execProcess(cmd))
   cache
 
-template uname(): untyped = unameRelease("uname -a", unameRes)
-template release(): untyped = unameRelease("lsb_release -a", releaseRes)
+template uname(): untyped = unameRelease("uname -o", unameRes)
```

Comment:
Why change this?

Reply:
`uname -a` and `lsb_release -a` contain the hostname, which can give false positives. For instance, my machine is named "anarchy" which contains the substring `arch` and gives a false positive for `Distribution.ArchLinux`